dev_guide/deployments.adoc

```diff
@@ -352,3 +352,21 @@ Rollbacks revert an application back to a previous deployment and can be
 performed using the REST API or the CLI. See the
 link:../cli_reference/basic_cli_operations.html#deployment-operations[CLI
 Reference] for more details.
+
+== Fine-Grained Control
+In addition to rollbacks, you can can exercise fine-grained control over
+the number of replicas by using the `osc resize` command.
+For example, the following command sets the replicas in the deployment
+configuration `widespread` to 3.
+
+----
+$ osc resize dc widespread 3
+----
+
+Doing this is equivalent to modifying the `widespread` deployment
+configuration and redeploying, but more straightforward when there are no
+other modifications to be done.
+
+[TIP]
+Use `osc resize` to quickly experiment with different scaling factors.
+Once you decide on a value, modify the configuration file, as well.
```

Comment:
I'm confused by this. A user shouldn't need to modify any configuration files when resizing.